appium-examples/Cucumber/emusim/features/support/env.rb

```diff
@@ -0,0 +1,38 @@
+require 'appium_lib'
+require 'sauce_whisk'
+require 'rspec'
+
+Before do | scenario |
+  # need to configure env variables for browser
+  caps = {
+      caps: {
+          platformVersion: "#{ENV['platformVersion']}",
```

Comment:
Ruby should convert snake case to camel case for these
Also, don't need quotes for these:
```
platformVersion: ENV['platformVersion']
```

Reply:
fixed